cdbot/cogs/fun.py

```diff
@@ -219,6 +219,15 @@ def check(reaction, user):
     async def on_reaction_add(self, reaction: Reaction, user: Member):
         if reaction.emoji == "\N{THUMBS DOWN SIGN}" and reaction.message.channel.id == QUOTES_CHANNEL_ID:
             if reaction.count >= 5:
+                conn = await asyncpg.connect(
+                    host=PostgreSQL.PGHOST,
+                    port=PostgreSQL.PGPORT,
+                    user=PostgreSQL.PGUSER,
+                    password=PostgreSQL.PGPASSWORD,
+                    database=PostgreSQL.PGDATABASE,
+                )
```

Comment:
Not necessarily in scope of this PR, but this snippet seems to be reused a couple times elsewhere. It might be worth setting up a [Connection Pool](https://magicstack.github.io/asyncpg/current/api/index.html?highlight=context#connection-pools) for the bot instance.

Reply:
#350